dll.py

```diff
@@ -1,46 +1,83 @@
 class Node(object):
 
-    def __init__(self, vertex: "Vertex", prev: "Vertex"=None, next: "Vertex"=None):
+    def __init__(self, vertex, prev=None, next_node=None):
+        """
+        Initializes a node
+
+        Creates a node with vertex as the node, and prev as the previous node and next as the next node in the list. If
+        no previous and next node are given, these values are set to `None`
+        :param vertex: node to add
+        :param prev: previous node
+        :param next_node: next node
+        """
         self._vertex = vertex
         self._prev = prev
-        self._next = next
+        self._next = next_node
```

Comment:
Waarom heb je deze aangepast en `prev` niet?

Reply:
Omdat python zeurde dat next een soort van beschermde term is

Reply:
alsnog zou ik het een soortgelijke naam houden